### PR TITLE
merge PR 416 from 1.0

### PR DESCRIPTION
**Pull request by dkent600**
Merges changes from https://github.com/daostack/arc.js/pull/416

### CODE REVIEW

File: src/plugins/proposal.ts [hunk 1]
```diff
@@ -158,7 +158,7 @@ export interface IProposalState {
 
 export abstract class Proposal<TProposalState extends IProposalState> extends Entity<
   TProposalState
-> {
+  > {
```
Comment (orenyodfat) [inline src/plugins/proposal.ts:161]: please remove the additional space here...

**Comment by orenyodfat** [comment]
these are just comments update.. right ? @dkent600

**Comment by dkent600** [comment]
> these are just comments update.. right ? @dkent600

yes

File: src/plugins/proposal.ts (hunk 1 above)
Comment (dkent600) [reply src/plugins/proposal.ts]: fixed

File: src/plugins/proposal.ts [hunk 2]
```diff
@@ -233,9 +231,9 @@ export abstract class Proposal<TProposalState extends IProposalState> extends En
           .join('\n')}
       }
       ${Object.values(Proposals)
-        .filter((proposal) => proposal.fragment)
-        .map((proposal) => proposal.fragment?.fragment.loc?.source.body)
-        .join('\n')}
+          .filter((proposal) => proposal.fragment)
```
Comment (orenyodfat) [inline src/plugins/proposal.ts:234]: space?

**Pull request merged**